### ISSUE TITLE
  tmpCssFile does not calculate a legal path on windows

### ISSUE THREAD

**Issue by WangShuXian6** [indooorsman/esbuild-css-modules-plugin#17]
>  tmpCssFile does not calculate a legal path on windows

version:2.1.2

![image](https://user-images.githubusercontent.com/30850497/153438808-f8eb9edc-713a-4d14-a240-9af1c8fbd0a4.png)
![image](https://user-images.githubusercontent.com/30850497/153438830-daac05fe-cc72-45b9-88b6-0eef8eaf2cb6.png)

```ts
      loader: {
        ".svg": "dataurl",
        ".png": "dataurl",
        ".jpg": "dataurl",
        ".jpeg": "dataurl",
        ".gif": "dataurl",
      },
      plugins: [
        nodeExternalsPlugin(),
        sassPlugin(),
        dtsPlugin(),
        cssModulesPlugin({
          inject: false,
          localsConvention: 'camelCaseOnly', // optional. value could be one of 'camelCaseOnly', 'camelCase', 'dashes', 'dashesOnly', default is 'camelCaseOnly'
          generateScopedName: (name, filename, css) => string, // optional. refer to: https://github.com/madyankin/postcss-modules#generating-scoped-names
          cssModulesOption: { 
            // optional, refer to: https://github.com/madyankin/postcss-modules/blob/d7cefc427c43bf35f7ebc55e7bda33b4689baf5a/index.d.ts#L27
            // this option will override others passed to postcss-modules
          },
    
          v2: true // experimental. v2 can bundle images in css, note if set `v2` to true, all other options will be ignored. and v2 only works with `bundle: true`.
        })
      ]
```

**Comment by WangShuXian6** [comment]
>To remove the `C:\User\WANGSH~1\` from the path,

> I replaced the code

```ts
  const tmpCssFile = path.join(
            tmpDir,
            fullPath.replace(rootDir, '').replace(/\.modules?\.css$/, '.modules_built.css')
          )
```

with

```ts
 const tmpCssFile = path
            .join(
              tmpDir,
              fullPath.replace(rootDir, '').replace(/\.modules?\.css$/, '.modules_built.css')
            )
            .replace(errorPath, '')
            .split(path.sep)
            .join(path.posix.sep)
```

> which solved the problem for now

**Comment by indooorsman** [comment]
could you paste the value of `fullPath` & `rootDir` here?  I do not have a Windows so it's not easy to debug

**Comment by indooorsman** [comment]
this issue should be fixed in v2.2.5, please upgrade to `v2.2.5` and try again with config `{v2: true}`
